mathesar_ui/src/App.svelte

```diff
@@ -19,16 +20,26 @@
     <Button on:click={() => newImport(selectedDb)}>
       Import CSV
     </Button>
+    <a href="schemas/">
```

Comment:
We could change this to 
```suggestion
    <a href="/{selectedDb}/schemas/">
```

Without a preceding slash, this would append `schemas/` to the current url. Clicking on this link within the schemas route will keep appending `schemas/` to the url.

Reply:
Committed the suggestion in 4bfe823.